require('sinon') throws FakeXMLHttpRequest error under NodeJS

```
mkdir newproj && cd newproj
npm init -y
npm i -S sinon@3.2.1
node -e 'require("sinon")'
```
Output:
```
/Users/thom/dev/voltserver/etx800/newproj/node_modules/just-extend/index.js:40
      throw new Error('expected object, got ' + arg);
      ^

Error: expected object, got function FakeXMLHttpRequest(config) {
    EventTargetHandler.call(this);
    this.readyState = FakeXMLHttpRequest.UNSENT;
    this.requestHeaders = {};
    this.requestBody = null;
    this.status = 0;
    this.statusText = "";
    this.upload = new EventTargetHandler();
    this.responseType = "";
    this.response = "";
    this.logError = configureLogError(config);

    if (sinonXhr.supportsTimeout) {
        this.timeout = 0;
    }

    if (sinonXhr.supportsCORS) {
        this.withCredentials = false;
    }

    if (typeof FakeXMLHttpRequest.onCreate === "function") {
        FakeXMLHttpRequest.onCreate(this);
    }
}
    at extend (/Users/thom/dev/voltserver/etx800/newproj/node_modules/just-extend/index.js:40:13)
    at Object.<anonymous> (/Users/thom/dev/voltserver/etx800/newproj/node_modules/nise/lib/fake-xhr/index.js:651:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/thom/dev/voltserver/etx800/newproj/node_modules/nise/lib/fake-server/index.js:3:15)
```
In fact this looks like an error in just-extend since a release was just published hours ago.

```
$ npm ls just-extend
newproj@1.0.0 /Users/thom/dev/voltserver/etx800/newproj
└─┬ sinon@3.2.1
  └─┬ nise@1.1.1
    └── just-extend@1.1.24 
$ node --version
v6.11.0
```

@angus-c 

+-- UNMET PEER DEPENDENCY chai@4.1.2
+-- UNMET PEER DEPENDENCY cheerio@0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.1
+-- UNMET PEER DEPENDENCY enzyme@3.1.0
+-- UNMET PEER DEPENDENCY eslint-plugin-jsx-a11y@6.0.2
+-- just-extend@1.1.22
+-- UNMET PEER DEPENDENCY popper.js@^1.12.3
+-- UNMET PEER DEPENDENCY react@16.0.0
+-- UNMET PEER DEPENDENCY react-dom@16.0.0
`-- UNMET PEER DEPENDENCY sinon@3.0.0

npm ERR! peer dep missing: chai@>=1.9.2 <4, required by sinon-chai@2.8.0
npm ERR! peer dep missing: enzyme@1.x || ^2.3.0, required by chai-enzyme@0.8.0
npm ERR! peer dep missing: enzyme@^2.0.0, required by enzyme-redux@0.1.7
npm ERR! peer dep missing: eslint-plugin-jsx-a11y@^5.1.1, required by eslint-config-airbnb@15.1.0
npm ERR! peer dep missing: react@^0.14.7 || ^15.0.1, required by react-autosuggest@8.0.1
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0, required by react-highlight-words@0.7.2
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0, required by recompose@0.24.0
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0-0, required by chai-enzyme@0.8.0
npm ERR! peer dep missing: react@^15.0.0, required by enzyme-redux@0.1.7
npm ERR! peer dep missing: react@^15.0.0, required by react-paginate@4.4.4
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0, required by react-window-resize-listener@1.1.0
npm ERR! peer dep missing: react@^0.14.0 || ^15.0.0, required by recharts@1.0.0-apha.5
npm ERR! peer dep missing: react@^0.14.9 || ^15.3.0, required by redux-devtools@3.4.0
npm ERR! peer dep missing: react@^0.14.0, required by window-resize-listener-react@1.0.3
npm ERR! peer dep missing: react-dom@^0.14.0 || ^15.0.0-0, required by chai-enzyme@0.8.0
npm ERR! peer dep missing: react-dom@^15.4.2, required by react-addons-test-utils@15.6.2
npm ERR! peer dep missing: sinon@>=1.4.0 <2, required by sinon-chai@2.8.0
npm ERR! peer dep missing: popper.js@^1.12.3, required by bootstrap@4.0.0-beta.2
npm ERR! peer dep missing: cheerio@0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.1, required by chai-enzyme@0.8.0

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "formatio": "1.2.0",
     "lodash.get": "^4.4.2",
     "lolex": "^2.1.3",
-    "nise": "^1.1.1",
+    "nise": "^1.2.0",
     "supports-color": "^4.4.0",
     "type-detect": "^4.0.0"
   },

